code/style.css

```diff
@@ -1,7 +1,212 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+}
 
-.active + .answer {
-  /* some styling here */
+/*** HEADER ***/
+
+header {
+  display: flex;
+  align-items: center;
+}
+
+video, img {
+  width: 100%;
+  filter: grayscale(100%); 
+} 
+
+@media (min-width: 1024px) { 
+  .img-tablet, .img-phone {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) { 
+  video, .img-phone {
+    display: none;
+  }
+}
+
+@media (max-width: 767px) { 
+  video, .img-tablet {
+    display: none;
+  }
+}
+
+h1 {
+  width: 100%;
+  position: absolute;
+  text-align: center;
+  color: white;
+  font-size: 22.1vw;
+
+  animation-name: lightning;
+  animation-duration: 0.1s;
+  animation-delay: 5s;
+  animation-iteration-count: 20;
+}
+
+@keyframes lightning {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  h1 {
+    animation-name: slowyellow;
+    animation-duration: 10s;
+    animation-delay: 2s;
+    animation-direction: alternate;
+    animation-iteration-count: infinite; 
+  }
+}
+
+@keyframes slowyellow {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (max-width: 767px) {
+  h1 {
+    color: yellow;
+    animation: none;
+  }
+}
+
+/*** CONTENT: SIGNUP FORM AND ACCORDION ***/
+
+.content-container {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  padding: 40px 20px 40px 20px;
```

Comment:
When the top and bottom have the same padding, and the left/right have the same - you could pass in two values instead of four (the first one applies to top/bottom and the second one to left/right):
`padding: 40px 20px;`

Reply:
Of course - thanks!